src/main/java/se/sundsvall/garbage/api/GarbageResource.java

```diff
@@ -2,30 +2,29 @@
 
 import java.util.List;
 
-import jakarta.validation.ConstraintViolation;
-import jakarta.validation.Valid;
-
 import org.springdoc.core.annotations.ParameterObject;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import org.zalando.problem.Problem;
-
-import se.sundsvall.garbage.api.model.GarbageScheduleRequest;
-import se.sundsvall.garbage.api.model.GarbageScheduleResponse;
-import se.sundsvall.garbage.service.GarbageService;
+import org.zalando.problem.violations.ConstraintViolationProblem;
 
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.media.Content;
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import se.sundsvall.garbage.api.model.GarbageScheduleRequest;
+import se.sundsvall.garbage.api.model.GarbageScheduleResponse;
+import se.sundsvall.garbage.service.GarbageService;
 
 @RestController
 @RequestMapping(value = "/schedules", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Remove `produces `here, since this will be "inherited" by the methods. Some methods does not produce APPLICATION_JSON_VALUE. It's probably better to define consumes/produces at method level?